auto_round/compressors/diffusion/dataset.py

```diff
@@ -0,0 +1,92 @@
+# Copyright (c) 2025 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+from typing import Dict
+
+import pandas as pd
+import torch
+from torch.utils.data import DataLoader, Dataset
+from transformers import set_seed
+
+from auto_round.utils import logger
+
+
+class DiffusionDataset(Dataset):
+    """Dataset for supervised fine-tuning."""
+
+    COCO_URL = {
+        "coco2014": "https://github.com/mlcommons/inference/raw/refs/heads/master/text_to_image/coco2014/captions/captions_source.tsv"
```

Comment:
do we need to download the all the coco files, is it too large?  I think Heng has already supported downloading part of  on VLM side , please check with him to refine the code here

Reply:
it is only a tsv file containing prompts and image id, it is 844K